app/models/application_record.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def formatted_unit_price
+    price = (unit_price.to_f / 100).round(2)
```

Comment:
nice workaround and if it saves you time in the future rails has a built in method for this `number_to_currency`